plugins/modules/hetzner_dns_record.py

```diff
@@ -54,7 +54,7 @@
 
 - name: Remove a new.foo.com A record
   community.dns.hosttech_dns_record_set:
```

Comment:
The used module is also wrong here.

Otherwise LGTM

Reply:
I have also corrected the module.